### PR TITLE
Added cron job to fetch chromium histograms

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This CL hopefully will make our life easier since we won't have to manually update UMA properties.

BUG=#97
TBR=@ebidel

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Please do not merge yet. Some speed improvements can be added.

**Comment by beaufortfrancois** [comment]
@ebidel The CL has been updated. 
Please have a look. It looks better now.

File: admin.py [hunk 1]
```diff
@@ -216,6 +220,67 @@ def _FetchFromBigstoreREST(self, filename):
...
+
+    if (result.status_code != 200):
+      logging.error('Unable to retrieve chromium histograms.')
+      return
+
+    browsed_histograms = []
+    histograms_content = result.content.decode('base64')
+    dom = xml.dom.minidom.parseString(histograms_content)
+
+    # The histograms.xml file looks like this:
```
Comment (ebidel) [inline admin.py]: There a raw source link we can point to?

File: admin.py [hunk 2]
```diff
@@ -216,6 +220,67 @@ def _FetchFromBigstoreREST(self, filename):
...
+    #   <int value="0" label="PageDestruction"/>
+    #   <int value="1" label="LegacyNotifications"/>
+
+    for enum in dom.getElementsByTagName('enum'):
+      histogram_id = enum.attributes['name'].value
+      if (histogram_id in self.MODEL_CLASS.keys()):
+        browsed_histograms.append(histogram_id)
+        for child in enum.getElementsByTagName('int'):
+          data = {
+            'bucket_id': child.attributes['value'].value,
```
Comment (ebidel) [inline admin.py]: needs a cast? `int(child.attributes['value'].value)`?

File: admin.py (hunk 1 above)
Comment (myakura) [inline admin.py]: adding `?format=TEXT` will take you to the raw source.

Comment (beaufortfrancois) [inline admin.py]: @ebidel Not sure what you mean. Do you want me to add https://chromium.googlesource.com/chromium/src/+/master/tools/metrics/histograms/histograms.xml?format=TEXT as a comment?

@myakura https://chromium.googlesource.com/chromium/src/+/master/tools/metrics/histograms/histograms.xml?format=TEXT returns the base64 source from what I can see

File: admin.py (hunk 2 above)
Comment (beaufortfrancois) [inline admin.py]: Integer cast is already done in `_SaveData`:

```javascript
bucket_id = int(data['bucket_id'])
```

Comment (ebidel) [inline admin.py]: Ack.

File: admin.py (hunk 1 above)
Comment (ebidel) [inline admin.py]: Yea, just adding a comment to the URL.

**Comment by ebidel** [comment]
LGTM. Just the URL pointer comment.

**Comment by beaufortfrancois** [comment]
URL pointer comment added.

**Comment by ebidel** [comment]
LGTM. Were you able to test this locally?

**Comment by beaufortfrancois** [comment]
I was and it worked fine.
I'm crossing my fingers that production server will say the same.

**Pull request merged**